deriv.com.conf

```diff
@@ -102,4 +102,33 @@ server {
     location ~* \.(pdf)$ {
         expires 5m;
     }
+    pagespeed on;
+
+    # Needs to exist and be writable by nginx.  Use tmpfs for best performance.
+    pagespeed FileCachePath /var/ngx_pagespeed_cache;
+
+    # Ensure requests for pagespeed optimized resources go to the pagespeed handler
+    # and no extraneous headers get set.
+    location ~ "\.pagespeed\.([a-z]\.)?[a-z]{2}\.[^.]{10}\.[^.]+" {
+     add_header "" "";
+    }
+    location ~ "^/pagespeed_static/" { }
+    location ~ "^/ngx_pagespeed_beacon$" { }
+    #charset koi8-r;
+
+    #access_log  logs/host.access.log  main;
+
```

Comment:
please commented lines that are related to configurations. you can move the module `on` to the upper side of the config as per the convention